modules/apps/blogs/blogs-editor-configuration/src/main/java/com/liferay/blogs/editor/configuration/internal/BlogsCoverImageCaptionEditorConfigContributor.java

```diff
@@ -55,13 +56,23 @@ public void populateConfigJSONObject(
 		String removePlugins = jsonObject.getString("removePlugins");
 
 		if (Validator.isNotNull(removePlugins)) {
-			removePlugins = removePlugins + ",magicline";
+			removePlugins = removePlugins + ",magicline,stylescombo,videoembed";
```

Comment:
`else` block is not covered.

Reply:
Yes. I added `,stylescombo,videoembed` as plugins to be removed there too :)